src/components/userProfile/components/UserInfo.js

```diff
@@ -1,49 +1,64 @@
 
-const { Meta } = Card;
-const {
-    CLASSNAMES: { CARD_META, DATA_TITLE, HEADER_TITLE, INFO_CARD },
-    TEXTS: { CONTACT_INFO_TEXT, LOCATION_INFO_TEXT },
+const { 
+    CLASSNAMES: { 
+        AVATAR, 
+        AVATAR_CONTAINER, 
+        EDIT_ICON, 
+        EDIT_BOTTON, 
+        INFO_CARD, 
+        INFO_DIV, 
+        ROW_CONTAINER, 
+        TITLE, 
+    },
+    PROFILE_PIX,
+    STRINGS: { EDIT, GHOST, SMALL, SQUARE },
+    TEXTS: { EDIT_PHOTO },
 } = USER_PAGE;
 
+// mapping various user information
+const profile = PROFILE_INFO.map(({ heading, info, key }) => ( 
+    <div key={key} className={INFO_DIV}>
+        <h4>
+            {heading}  
+            <Icon type={EDIT} className={EDIT_ICON} />
+        </h4>
+        {
+            info.map(({ description, title }) => (
+                <Row gutter={16} key={title} className={ROW_CONTAINER}>
+                    <Col span={12} className={TITLE}>{title}</Col>
+                    <Col span={12}>{description}</Col>
+                </Row>
+            ))
+        }
+    </div>
+
+));
+
+// react component used to render user information
 export default class UserInfo extends React.Component {
     render() {
+        
         return (
             <Card
                 className={INFO_CARD}
                 bordered={false}
             >
-                <Meta
-                    title={<h4 className={HEADER_TITLE}>{CONTACT_INFO_TEXT}</h4>}
-                    description={
-                        CONTACT.map(contact => (
-                            <div key={contact.title} className={DATA_TITLE}>
-                                <b>{contact.title}</b>
-                                <div>
-                                    {contact.description}
-                                </div>
-                            </div>
-                        ))
-                    }
-                />
-                <Divider />
-                <Meta
-                    className={CARD_META}
-                    title={<h4 className={HEADER_TITLE}>{LOCATION_INFO_TEXT}</h4>}
-                    description={
-                        LOCATION.map(location => (
-                            <div key={location.title} className={DATA_TITLE}>
-                                <b>{location.title}</b>
-                                <div>
-                                    {location.description}
-                                </div>
-                            </div>
-                        ))
-                    }
-                />
+                <div className={AVATAR_CONTAINER}>
+                    <Avatar
+                        className={AVATAR}
+                        src={PROFILE_PIX}
+                        shape={SQUARE}
+                    />
+                    <Button type={GHOST} size={SMALL} className={EDIT_BOTTON}>
+                        {EDIT_PHOTO}
+                        <Icon type={EDIT} />
+                    </Button>
```

Comment:
![image](https://user-images.githubusercontent.com/34488930/58753056-1ea94b80-84b1-11e9-9bec-eff10586c0ef.png)
I actually think the edit button will look better as just an icon on top of the image maybe by the right or just beside the image

Reply:
I think its ok where it is